casepro/statistics/migrations/0001_initial.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+from django.conf import settings
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('orgs', '0016_taskstate_is_disabled'),
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('cases', '0037_partner_is_restricted'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='DailyOrgCount',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
+                ('type', models.CharField(max_length=1)),
+                ('day', models.DateField(help_text='The day this count is for')),
+                ('count', models.PositiveIntegerField(default=1)),
+                ('squashed', models.BooleanField(default=False)),
+                ('org', models.ForeignKey(to='orgs.Org')),
+            ],
+            options={
+                'abstract': False,
+            },
+        ),
+        migrations.CreateModel(
+            name='DailyUserCount',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
+                ('type', models.CharField(max_length=1)),
+                ('day', models.DateField(help_text='The day this count is for')),
+                ('count', models.PositiveIntegerField(default=1)),
+                ('squashed', models.BooleanField(default=False)),
+                ('org', models.ForeignKey(to='orgs.Org')),
+                ('user', models.ForeignKey(to=settings.AUTH_USER_MODEL)),
+            ],
+            options={
+                'abstract': False,
+            },
+        ),
+        migrations.CreateModel(
+            name='DailyPartnerCount',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
+                ('type', models.CharField(max_length=1)),
+                ('day', models.DateField(help_text='The day this count is for')),
+                ('count', models.PositiveIntegerField(default=1)),
+                ('squashed', models.BooleanField(default=False)),
+                ('partner', models.ForeignKey(to='cases.Partner')),
+            ],
```

Comment:
Playing devil's advocate, but are there advantages to splitting these out by org / user / partner vs having that encoded in the 'type' (say as a slug?)

Don't really know the answer here, just throwing it out there. @norkans7  went the route of essentially building slugs that represented what he wanted to keep count of from org_ids or user_ids etc.. Which leads to a single model that is maintained for counts etc.. 

Perhaps he can comment on the pros and cons there.